uint8 hMin
uint8 hMax
uint8 sMin
uint8 sMax
uint8 vMin
uint8 vMax
